src/app/orbit/services/airtable.ts

```diff
@@ -12,22 +13,28 @@ const airtableRecordSchema = z.object({
   [TITLE]: z.string(),
   [CITY]: z.string(),
   [START_DATE]: z.coerce.date(),
-  [REGISTRATION_LINK]: z.string().url(),
+  [REGISTRATION_LINK]: z.string().url().optional(),
 })
 
-const airtableRecordsSchema = z.array(airtableRecordSchema)
-
 export async function fetchAndParseAirtableEvents() {
-  const rawAirtableRecords = await airtable
+  const rawAirtableRecords = await fetchAirtableRecords()
+  const validatedRecords = validateRawRecords(rawAirtableRecords)
+  const humanReadableRecords = getHumanReadableRecords(validatedRecords)
+
+  return humanReadableRecords
+}
+
+function fetchAirtableRecords() {
+  return airtable
     .base(BASE_ID)
     .table(EVENTS_TABLE_ID)
     .select({
       fields: Object.values(FIELDS),
       returnFieldsByFieldId: true,
       filterByFormula: `AND(
-        {${FIELDS.STATUS}} = '${APPROVED_STATUS_VALUE}',
-        IS_AFTER({${FIELDS.START_DATE}}, '${new Date().toISOString()}')
-      )`,
+      {${FIELDS.STATUS}} = '${APPROVED_STATUS_VALUE}',
+      IS_AFTER({${FIELDS.START_DATE}}, '${new Date().toISOString()}')
```

Comment:
Can this need to be reworked `'${new Date().toISOString()}'` given that we now have `[START_DATE]: z.coerce.date()`?